host_core/test/test_helper.exs

```diff
@@ -1,3 +1,4 @@
 Code.require_file("support/event_watcher.exs", __DIR__)
+Code.require_file("support/constants.exs", __DIR__)
```

Comment:
There's a way to get around this and not need to use the `test_helper.exs` file in order to gain access to these files. No need to fix this now, I'll probably clean that up in a subsequent PR.